### ISSUE TITLE
HW0 & HW1

### ISSUE THREAD

**Issue by wanitchayap** [Computational-Content-Analysis-2020/frequently-asked-questions-spring#14]
Would HW for week 0 be graded too?

Also for HW1, in the NER section, the prompt asks us to calculate F1, recall, and precision. However, we do not necessarily have gold labels for our corpus...?

Thank you!

**Comment by bhargavvader** [comment]
Yes, HW0 will also be graded (as one of the 5 HWs.) You are expected to either find a dataset which has labels, or just use a small sample and manually add your own labels. This is to just demonstrate those concepts, so a small proof of concept code is enough.

**Comment by DSharm** [comment]
I'm struggling to calculate the F1, recall, and precision. Was this described somewhere in the readings / the notebook? Apologies if I missed it!
